docs/source/API/core/initialize_finalize/ScopeGuard.rst

```diff
@@ -17,21 +17,20 @@ Usage
                                 .set_num_threads(1));
 
 
-``ScopeGuard`` is a class which ensure thats `Kokkos::initialize <initialize.html#kokkosinitialize>`_ and
-`Kokkos::finalize <finalize.html#kokkosfinalize>`_ are called correctly even in the presence of unhandled
-exceptions.
+``ScopeGuard`` is a class to initialize and finalize Kokkos using RAII.
```

Comment:
Add a reference to https://en.cppreference.com/w/cpp/language/raii from "RAII"

Reply:
good idea, done in [14099c](https://github.com/kokkos/kokkos-core-wiki/pull/575/commits/6b5b3eee663b71cf4e3af1ae0d2800c7c314099c)